### ISSUE TITLE
Collision Hero Circle 

### ISSUE THREAD

**Issue by Tony-ATC** [Oddebit/ShootBadGuys#3]
Le "Collision Circle" du hero est un peu trop grand, je suis passe juste a côté des ennemies plusieurs fois sans les toucher mais m'a quand même fait perdre des HP.

- Reduire le cercle de collision du Hero

**Comment by Oddebit** [comment]
oui, pcq le collision bound est carré, je dois encore changer ca

**Comment by Tony-ATC** [comment]
Je crois qui si ont essaye RoundRectangle2D ça va aider la situation non?

**Comment by Oddebit** [comment]
ya la method intersects?

**Comment by Oddebit** [comment]
toute facon, je pense que je vais faire la méthode intersect() moi-même...

**Comment by Tony-ATC** [comment]
Normalement oui vue qu'elle fait partie du interfaces Shape

The RoundRectangle2D class defines a rectangle with rounded corners defined by a location (x,y), a dimension (w x h), and the width and height of an arc with which to round the corners.
This class is the abstract superclass for all objects that store a 2D rounded rectangle. The actual storage representation of the coordinates is left to the subclass.
